tests/main/netplan-apply/task.yaml

```diff
@@ -6,12 +6,18 @@ details: |
 environment:
     NETPLAN: io.netplan.Netplan
 
-# run on all classic ubuntu LTS and current dev systems 16.04+
-systems: 
-    - ubuntu-16*
-    - ubuntu-18*
-    - ubuntu-19*
-    - ubuntu-20*
+# don't run on non-ubuntu systems where we won't have netplan
+# and also don't run on ubuntu-core where we can't create the fake netplan D-Bus
+# service
+systems:
```

Comment:
I actually switched to a blacklist as your suggestion in the original PR here: https://github.com/snapcore/snapd/pull/7214/files#r310892524

It would be nice if there was some way to specify "not ubuntu" as a blacklist then it would just become something like:

```yaml
systems:
  - -"not-ubuntu"
  - -ubuntu-core-*
  - -ubuntu-14*
```

Reply:
Ah, funny this :)